src/components/post/Post.jsx

```diff
@@ -1,12 +1,13 @@
-import styles from '@/components/post/index.module.scss'
+import styles from "@/components/post/index.module.scss";
+import Link from "next/link";
 
 const Post = ({ className, content, title }) => {
   return (
-    <section className={className}>
+    <Link href={`/${title}`} className={className}>
```

Comment:
@paolospoto qui bisognerebbe avere il path: `/posts`

```suggestion
    <Link href={`/posts/${title}`} className={className}>
```